README.md

```diff
@@ -1,24 +1,162 @@
 
-* Database initialization
 
-* How to run the test suite
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/documentation/">Ruby on Rails</a></li>
+    <li><a href="https://www.ruby-lang.org/en/documentation/">React</a></li>
+  </ul>
 
-* Services (job queues, cache servers, search engines, etc.)
 
-* Deployment instructions
+<!-- Features -->
 
-* ...
+### Key Features <a name="key-features"></a>
+
+- **Users can view a random welcome message**
+- **Uses rails backend**
+- **Uses react front end**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Key Features <a name="key-features"></a>
+  
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get the content of this project locally you need to run the following commands in your terminal:
+
+and follow these steps.
+
+### Prerequisites
+
+- IDE(code editor) like: **Vscode**, **Sublime**, etc.
+- [Git](https://www.linode.com/docs/guides/how-to-install-git-on-linux-mac-and-windows/)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+1. Download the **Zip** file or clone the repo with:
+ ```bash
+git clone git@github.com:gjuliao/hello_rails_react.git
+```
+2. To access cloned directory run:
+```bash
+cd recipe_app
+```
+
+### Install
+
+Install dependancies with:
+
+```bash
+bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```bash
+rails s
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+rspec spec
+```
+
+### Deployment
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Giovanni Juliao**
+
+- GitHub: [@gjuliao](https://github.com/gjuliao)
+- LinkedIn: [giovanni-juliao](https://www.linkedin.com/in/giovanni-juliao/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Add more welcome messages **
+- **Make a mobile version app**
+- **Authenticate user email with Devise**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+
+Feel free to check the [issues page](https://github.com/gjuliao/hello_rails_react/issues).
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+The microverse team for organizing this guiding project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
* Kindly fix your MIT link it is currently broken

Reply:
Hi Davidosky007, the license works in the dev branch.  In order for it to work it needs to be merged with dev.